src/main/flight/pid_init.c

```diff
@@ -81,25 +82,50 @@ void pidInitFilters(const pidProfile_t *pidProfile)
 
     const uint32_t pidFrequencyNyquist = pidRuntime.pidFrequency / 2; // No rounding needed
 
-    uint16_t dTermNotchHz;
-    if (pidProfile->dterm_notch_hz <= pidFrequencyNyquist) {
-        dTermNotchHz = pidProfile->dterm_notch_hz;
+    //1st Dterm Notch Filter
+
+    uint16_t dTermNotch1Hz;
+    if (pidProfile->dterm_notch1_hz <= pidFrequencyNyquist) {
+        dTermNotchHz = pidProfile->dterm_notch1_hz;
```

Comment:
```suggestion
        dTermNotch1Hz = pidProfile->dterm_notch1_hz;
```

Reply:
I think making them 1 and 2 is the generalised approach also used for gyro notch and low passes...